native/src/hooks/useTtsPlayer.ts

```diff
@@ -0,0 +1,32 @@
+import { useContext, useEffect } from 'react'
+
+import { ttsContext, TtsContextType } from '../components/TtsPlayer'
+
+const useTtsPlayer = (content?: string, title?: string): TtsContextType => {
```

Comment:
:thinking: How about extracting the sentence creation logic to here? This way, we would not need duplicated state (title, content, sentences) and using this would be more easy.
```ts
const useTtsPlayer = (model?: PageModel | LocalNewsModel): TtsContextType => {
  const sentences = useMemo(
    () => {
      if (model) {
        const content = parseHTML(model.content) 
        return [model.title, ...segment(languageCode, content)]
      }
      return null
    },
    [model, languageCode],
  )

  useEffect(() => {
    tts.setSentences(sentences)
    return () => {
      tts.setSentences([])
    }
  }, [sentences, tts])
```